alembic/versions/625ae864b811_add_password_column_to_users.py

```diff
@@ -0,0 +1,28 @@
+"""add password column to users
+
+Revision ID: 625ae864b811
+Revises: 94d6fd658341
+Create Date: 2024-04-28 21:01:49.536618
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+from app import utils
+
+# revision identifiers, used by Alembic.
+revision: str = '625ae864b811'
+down_revision: Union[str, None] = '94d6fd658341'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.add_column('users', sa.Column('password', sa.String(), nullable=True))
+    op.execute(f"UPDATE users SET password = '{utils.hash('1234')}' WHERE password IS NULL")
```

Comment:
That's actually showing the default password to the entire world. Might be the security thread. Probably it's better to set actual hash - it is not available for backward decription